Nef_3/include/CGAL/Nef_polyhedron_3.h

```diff
@@ -164,8 +164,8 @@ class Nef_polyhedron_3 : public CGAL::Handle_for< Nef_polyhedron_3_rep<Kernel_,
   typedef typename Kernel::Segment_3                  Segment_3;
   typedef typename Kernel::Aff_transformation_3       Aff_transformation_3;
 
-#ifndef _MSC_VER
-  // VC++ has a problem to digest the following typedef,
+#if _MSC_VER >= 1900
```

Comment:
The patch is still wrong: the code block must be compiled if the compiler is not MSVC.

Maybe:
```c++
#if ! (_MSC_VER < 1900)
```

Reply:
No, that proposal would be wrong as well. We should test:
```C++
#if (! defined _MSC_VER) || (_MSC_VER >= 1900)
```

Reply:
Right, I'm making the change